.circleci/config.yml

```diff
@@ -236,8 +236,9 @@ jobs:
                   corresponding_milestone_title=v$(extract_version_from_current_wd)
                   tag=$corresponding_milestone_title
                   change_notes=$(./gradlew getChangelog --no-header -q)
+                  signed_plugin_zip_path=$(./gradlew -q printPluginZipPath | sed 's/.zip/-signed.zip/')
```

Comment:
This name format is brittle, might change with subsequent release.

I'd rather add a separate `printSignedPluginZipPath`.

Also, how about
> There's one catch: the name of the output zip from signPlugin seems to differ depending on whether the zip file exists.
If a zip file already exists, a new file with -signed suffix is created; if not, then the signed zip is saved under the base name.
To be researched if there are any traps here that could affect any of the contexts.

❓

Reply:
Both unsigned and signed plugin zips should be there already, as we have the `./gradlew publishPlugin` invoked earlier. At least this is what we have in CONTRIBUTING.md so it should be true:
> Then run `./gradlew publishPlugin` to produce the unsigned and signed `.zip` files in the `build/distributions/` directory.

So, this problem should not be the case here.

<img width="1356" alt="image" src="https://user-images.githubusercontent.com/49597713/193028346-67b1dab1-0daa-45bf-8489-6cf034ad5bc3.png">